lib/index.js

```diff
@@ -77,13 +76,14 @@ function getLoaderConfig (ctx) {
     includePaths: [],
     data: '',
     transformers: [],
-    outputStyle: 'nested',
-    resolveURLs: true
+    resolveURLs: true,
+    sassOptions: {}
```

Comment:
to avoid break change, you should add compatible code for `outputStyle `, make sure the old way can still work

Reply:
I've explained why I needed to make that change in the "Caveat" section of the PR's description. Would it be an issue for you to publish a major version of this package after this PR is merged?